### PR TITLE
Feature 280224 sj category landing

### PR DESCRIPTION
**Pull request by MTechSanjeev**
### Problem
- category landing page
### Preview Link: https://parkavenue-india.myshopify.com/collections/landing-collection
- created category landing page
### Impacted Areas
- N/A
### Done Checklist
- [x] Performed unit testing to ensure functionality and identify potential issues.
- [x] Followed basic code standards and best practices. (indentation, media queries, basic customizer things)
- [x] Added comments to explain complex logic or improve code readability.
- [x] Followed Naming conventions and proper variable declaration.
- [x] Handled edge cases and errors gracefully to prevent crashes.
- [x] Optimized the code for production to ensure performance.
Please review this PR and provide feedback. Thank you!

### CODE REVIEW

File: assets/home-custom.css [hunk 1]
```diff
@@ -188,4 +188,204 @@
...
+  width: 218px;
+  height: 303px;
+}
+
+.park_shop_the_look .hotspot_second_content {
+  position: relative;
+  width: 218px;
+  height: 303px;
+}
+
```
Comment (S-Darshan) [inline assets/home-custom.css:336]: css repeating.

File: assets/home-custom.css [hunk 2]
```diff
@@ -188,4 +188,204 @@
...
+  padding: 18px 13px 0px 13px;
+  background-color: #FFF;
+}
+
+.park_shop_the_look .second_popup_box {
+  position: absolute;
+  top: 0px;
+  right: 0px;
+  padding: 18px 13px 0px 13px;
+  background-color: #FFF;
```
Comment (S-Darshan) [inline assets/home-custom.css:350]: css repeating.

File: assets/home-custom.css [hunk 3]
```diff
@@ -188,4 +188,204 @@
...
+  width: 189px;
+  height: auto;
+  aspect-ratio: 0.84;
+}
+
+.park_shop_the_look .second_popup_pic img {
+  width: 189px;
+  height: auto;
+  aspect-ratio: 0.84;
+}
```
Comment (S-Darshan) [inline assets/home-custom.css:363]: css repeating.

Comment (MTechSanjeev) [reply assets/home-custom.css:363]: css repeating. >>>>>>>>>> repeatation removed

File: assets/home-custom.css (hunk 2 above)
Comment (MTechSanjeev) [reply assets/home-custom.css:350]: css repeating. >>>>>>>>>> repeatation removed

File: assets/home-custom.css (hunk 1 above)
Comment (MTechSanjeev) [reply assets/home-custom.css:336]: css repeating. >>>>>>>>>> repeatation removed

**Pull request merged**